src/cbor.c

```diff
@@ -26,7 +26,7 @@
 #include <assert.h>
 #include <dps/private/cbor.h>
 
-#if CBOR_MAX_STRING_LEN >= RSIZE_MAX_STR
+#if CBOR_MAX_STRING_LEN > RSIZE_MAX_STR
 #error CBOR_MAX_STRING_LEN must be less than RSIZE_MAX_STR (see safe_str_lib.h)
```

Comment:
With your change this is now "less than or equal" is that correct - terminated NUL not included in the size?

Reply:
I'll double check this one